boxsdk/object/terms_of_service_user_status.py

```diff
@@ -0,0 +1,39 @@
+# coding: utf-8
+
+from __future__ import unicode_literals
+from functools import partial
+import json
+
+from ..config import API
+from .base_object import BaseObject
+from ..util.translator import Translator
+
+
+class TermsOfServiceUserStatus(BaseObject):
```

Comment:
You can probably put this into terms_of_service.py, rather than creating a new file for it.